src/table/hooks/use-focus-listener.ts

```diff
@@ -1,12 +1,17 @@
+import { stardust } from '@nebula.js/stardust';
 import { useEffect } from 'react';
 import { handleFocusoutEvent } from '../utils/handle-accessibility';
 
-const useFocusListener = (tableWrapperRef, shouldRefocus, keyboard) => {
+const useFocusListener = (
+  tableWrapperRef: React.MutableRefObject<HTMLElement>,
```

Comment:
i prefer to use HTMLDivElement

Reply:
yeah both seem to be divs so we can use that on both container and wrapper